Packs/PANOSPolicyOptimizer/pack_metadata.json

```diff
@@ -2,8 +2,8 @@
     "name": "PAN-OS Policy Optimizer",
     "description": "This integration introduces Policy Optimizer and DAG features that are not available through the regular PAN API",
     "support": "xsoar",
-    "currentVersion": "1.0.3",
-    "author": "Cortex XSOAR",
+    "currentVersion": "1.0.4",
+    "author": "Maciej Drobniuch and Cortex XSOAR",
```

Comment:
I am not sure what will happen in the upload process in this case as currently, XSOAR supported paks should have the author as cortex xsoar.

Reply:
IIRC, it should be fine